Observing-Interface-Setup/dds-interface.rst

```diff
@@ -11,7 +11,7 @@ There are 3 use-cases for DDS Daemons defined by their location and application.
 #. Local daemons for local CSCs or services which are mostly used for non-production environments (i.e. development)
 #. Nublado daemons which are mainly used as a way to manually ensure system stability for night time operations.
 
-Prerequisites
+Precondition
 ^^^^^^^^^^^^^
```

Comment:
^^^^^ --> =====?

Reply:
Have replaced it in some other docs in the same folder.